jukebox/src/main/webapp/js/add_song.js

```diff
@@ -0,0 +1,54 @@
+var CLIENT_ID = "2440856049-ga26c9i6ijv84jsedvc0ddsqi1bk1ftd.apps.googleusercontent.com";
+var API_KEY = "AIzaSyBTz_m7PhhcCWy1RxbMb1O24ItNAoRN6S0";
```

Comment:
It'd be a good idea to hide this (as well as the CLIENT_ID)
see here: https://cloud.google.com/docs/authentication/api-keys#securing_an_api_key

(It'd be a good idea to regenerate the API key now as well)

Reply:
I originally had it to where it was resticted to appspot.com. I just regenerated the API key and posted it to our team chat

Reply:
that means we can get rid of these two lines here right?